ui/scripts/ui-custom/instanceWizard.js

```diff
@@ -979,6 +988,8 @@
 
                                     $vpcSelect.val(-1);
 
+                                    sortArrayByKey(args.data.networkOfferings, 'name');
```

Comment:
Network offerings also has a support for sort_key param in database. We should not sort this.

Reply:
I had missed this earlier in my list. Apologies for that.